src/app/EmployeesHistory/Components/employeeSummary.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import ItemsList from './itemsList';
+import EmployeeHistoryProvider from '../employeeHistoryProvider';
+
+class EmployeeSummary extends Component {
+  state = {
```

Comment:
Thumbs-up for initialising the components state this way rather than in constructor.